command-line/lib/Command/Key/Hash.hs

```diff
@@ -68,13 +70,16 @@ run Hash = do
                 fail "data should be a 32-byte public key."
 
     prefixFor hrp
-        | hrp == CIP5.addr_vk    = CIP5.addr_vkh
-        | hrp == CIP5.addr_xvk   = CIP5.addr_vkh
+        | hrp == CIP5.addr_vk          = CIP5.addr_vkh
+        | hrp == CIP5.addr_xvk         = CIP5.addr_vkh
 
-        | hrp == CIP5.stake_vk   = CIP5.stake_vkh
-        | hrp == CIP5.stake_xvk  = CIP5.stake_vkh
+        | hrp == CIP5.stake_vk         = CIP5.stake_vkh
+        | hrp == CIP5.stake_xvk        = CIP5.stake_vkh
 
-        | hrp == CIP5.script_vk  = CIP5.script_vkh
-        | hrp == CIP5.script_xvk = CIP5.script_vkh
+        | hrp == CIP5.shared_addr_vk   = CIP5.script_vkh
+        | hrp == CIP5.shared_addr_xvk  = CIP5.script_vkh
+
+        | hrp == CIP5.shared_stake_vk  = CIP5.script_vkh
+        | hrp == CIP5.shared_stake_xvk = CIP5.script_vkh
```

Comment:
I'd be in favor of dropping the `script_` prefix to use something more consistent like `addr_shared_vkh` and `stake_shared_vkh`

Reply:
So, we would also rename `shared_addr_vk` to `addr_shared_vk` to match, right?

I pushed a suggestion to use a mapping of prefixes instead of conditionals.
It makes the code shorter.
Same could be applied in the other CLI commands.

Reply:
> So, we would also rename shared_addr_vk to addr_shared_vk to match, right?

Yes. Sorry, we originally hinted towards `shared_thing` yesterday, but I found the `thing_shared` order a bit more helpful / readable. So, there will be some massive `sed` to do in the end.

Reply:
will do it in the coming commit